spec/features/embedded_apps_spec.rb

```diff
@@ -1,10 +1,10 @@
 require 'rails_helper'
 
-RSpec.feature "Embedded apps", type: :feature do
-  let!(:journey) {create(:journey)}
-  let!(:child_birth_step) {create(:step, journey: journey, description: '<embedded-app app-id="narodenie-rodny-list" />')}
+RSpec.feature "Embedded apps" do
+  let!(:journey) { create(:journey) }
+  let!(:child_birth_step) { create(:step, journey: journey, description: '<embedded-app app-id="narodenie-rodny-list" />') }
 
-  scenario 'As a user I want to see an the Child Birth Picking up Protocol app' do
+  scenario 'inserts the Child Birth Picking up Protocol app' do
```

Comment:
Toto prosim nechaj ako to bolo. Je to take ukecane, ale je to [user story](https://www.mountaingoatsoftware.com/agile/user-stories).